lib/controllers/books.js

```diff
@@ -0,0 +1,23 @@
+const { Router } = require('express');
+const Book = require('../models/Book');
+
+module.exports = Router()
+  .get('/:id', async (req, res) => {
+    const book = await Book.getById(req.params.id);
```

Comment:
Currently, you're getting a book back from your getById method - so, if there was a method that aggregated your authors (by book ID), it could be called here, and assigned book object (same idea for your authors controller!)